pkg/services/app/add_test.go

```diff
@@ -472,22 +472,6 @@ var _ = Describe("Add", func() {
 
 			Expect(filters[0](".wego/file.txt")).To(BeTrue())
 		})
-
-		It("creates a pr with branch hash name", func() {
-			defaultParams.AutoMerge = false
-			err := appSrv.Add(defaultParams)
-			Expect(err).ShouldNot(HaveOccurred())
-
-			_, _, newBranch, files, _, _, _ := gitProviders.CreatePullRequestToUserRepoArgsForCall(0)
-			Expect(newBranch).To(Equal("wego-bf22e886ea99b1891c16bba5529d7f0b"))
-			Expect(files).To(Not(BeEmpty()))
-
-			Expect(gitClient.WriteCallCount()).To(Equal(0))
-			Expect(kubeClient.ApplyCallCount()).To(Equal(2))
-			Expect(fluxClient.CreateSecretGitCallCount()).To(Equal(1))
-			Expect(gitClient.CloneCallCount()).To(Equal(0))
-			Expect(gitProviders.CreatePullRequestToUserRepoCallCount()).To(Equal(1))
-		})
```

Comment:
These tests cant be run now that createrepo is returning the url object. These tests are still covered in common_test.go

Reply:
So the test cant get the url from the url object because it was never set. Before it never did anything with the url.  Now it errors out in add.go because that stuff doesnt exist. Anyway i realized these were duplicate tests anyway and are checked in acceptance and common_test.